### PR TITLE
Add schedule and git tutorial template

### PR DESCRIPTION
**Pull request by JameelB**
## Description
Generate changes for Week 12 Schedule and the addition of the git tutorial templates. 

## Additional Notes: 
Changes are based on the following PRs
- https://github.com/feedhenry/student-help-guide-content/pull/5
- https://github.com/feedhenry/student-help-guide-content/pull/3

### CODE REVIEW

**Review by damienomurchu — APPROVED** [review]
Pulled and viewed code - looks good!

File: topic04-git-introduction/book-git-tutorial/indexchat.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
```
Comment (laurafitzgerald) [inline topic04-git-introduction/book-git-tutorial/indexchat.html]: not sure about this file? seems to set an `Open Chat` button which just opens an empty section

File: topic04-git-introduction/index.html [hunk 2]
```diff
@@ -173,6 +177,30 @@ <h2>  Git Introduction </h2>
 </div>
 
           
+
```
Comment (laurafitzgerald) [inline topic04-git-introduction/index.html:166]: What's the difference between index.html and indexmodule.html

**Review by laurafitzgerald — COMMENTED** [review]
some questions inline @JameelB

File: topic04-git-introduction/index.html (hunk 2 above)
Comment (JameelB) [reply topic04-git-introduction/index.html:166]: @laurafitzgerald It seems like the index.html contains all the contents of the page and the indexmoodle.html seems to contain the contents of the card.

File: topic04-git-introduction/book-git-tutorial/indexchat.html (hunk 1 above)
Comment (JameelB) [reply topic04-git-introduction/book-git-tutorial/indexchat.html]: @laurafitzgerald That's actually in all sections. It's a generated file. We can remove it if needed. It seems to direct to gitter

File: topic04-git-introduction/index.html (hunk 2 above)
Comment (laurafitzgerald) [reply topic04-git-introduction/index.html:166]: @JameelB thanks for that.

File: topic04-git-introduction/book-git-tutorial/indexchat.html (hunk 1 above)
Comment (laurafitzgerald) [reply topic04-git-introduction/book-git-tutorial/indexchat.html]: @JameelB I think we should remove it as it opens up only an empty panel.

**Comment by JameelB** [comment]
@laurafitzgerald just found out that the chat seems to be embedded within the index pages itself as a script. Even if we remove the indexchat files, the `open chat` button is still going to be there.

**Comment by laurafitzgerald** [comment]
@JameelB did some investigation and removing the script removes the panel. Pushed up a commit. Could you verify when you get a chance?

**Comment by JameelB** [comment]
@laurafitzgerald that would remove the gitter chat panel however once it's generated again with new changes, it'll just override it and add those in again.

**Comment by laurafitzgerald** [comment]
@JameelB hmm okay. Looks like they are gone now so let's get this one merged anyway. And think about how we manage this going forward

**Pull request merged**